packages/shared/src/components/modals/OnboardingModal.tsx

```diff
@@ -127,6 +138,15 @@ function OnboardingModal({
     }
   };
 
+  useEffect(() => {
+    trackEvent({
+      event_name: 'impression',
```

Comment:
I think we should use enums for the values

Reply:
Totally agree with both of you.